cmd2/utils.py

```diff
@@ -509,11 +515,11 @@ def line_buffering(self) -> bool:
         when running unit tests because pytest sets stdout to a pytest EncodedFile object.
         """
         try:
-            return self.inner_stream.line_buffering
+            return bool(self.inner_stream.line_buffering)
```

Comment:
Isn't line_buffering already a bool?

Reply:
Apparently it's `int` on one of the versions of python we're using.